### ISSUE TITLE
[v1.6] Incorrect placement

### ISSUE THREAD

**Issue by GB-CG** [Sappharad/MultiPatch#3]
When I tried to place a patch file, it goes to the 'Location of file to patch' instead of 'Location of patch file'. And it's the other way round when I put a game file?
<img width="563" alt="screen shot 2017-04-23 at 17 48 38" src="https://cloud.githubusercontent.com/assets/11931125/25316063/e056c07c-2857-11e7-9b32-99a517225bcf.png">

**Comment by Sappharad** [comment]
Yikes. Good thing I hadn't submitted 1.6 to RHDN yet.

To add drag and drop in 1.6, I had to reorganize some stuff and reconnect the Browse buttons to the code after that. I did a poor job naming the buttons internally, so I guess I got confused and hooked them up in the wrong order. I made the same mistake with the drag and drop too, but caught that in testing because it was a new feature yet I neglected to re-test the old stuff that was working before.

Sorry about that, will release a new version tonight.
